### PR TITLE
update zora chain details

### PR DESCRIPTION
**Pull request by jackiehluo**
## Description

updates the name and rpc/block explorer urls for zora network.

## Additional Information

- [✅] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

### CODE REVIEW

**Review by tmm — COMMENTED** [review]
Please add a changeset with `pnpm changeset`.

**Comment by jackiehluo** [comment]
updated with changeset!

File: .changeset/orange-waves-shake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@wagmi/chains": patch
+---
+
+Update Zora chain URLs and names
```
Comment (tmm) [inline .changeset/orange-waves-shake.md:5]:
```suggestion
Updated Zora chain URLs and names
```

File: packages/chains/src/zoraTestnet.ts [hunk 2]
```diff
@@ -2,26 +2,29 @@ import { Chain } from './types'
...
+      webSocket: ['wss://testnet.rpc.zora.energy'],
     },
   },
   blockExplorers: {
-    etherscan: { name: 'ZORA', url: 'https://testnet.explorer.zora.co' },
-    default: { name: 'ZORA', url: 'https://testnet.explorer.zora.co' },
+    etherscan: {
+      name: 'Explorer',
+      url: 'https://testnet.explorer.zora.energy',
+    },
```
Comment (tmm) [inline packages/chains/src/zoraTestnet.ts:26]:
```suggestion
```

File: packages/chains/src/zora.ts [hunk 3]
```diff
@@ -11,17 +11,17 @@ export const zora = {
...
-      http: ['https://rpc.zora.co'],
-      webSocket: ['wss://rpc.zora.co'],
+      http: ['https://rpc.zora.energy'],
+      webSocket: ['wss://rpc.zora.energy'],
     },
   },
   blockExplorers: {
-    etherscan: { name: 'ZORA', url: 'https://explorer.zora.co' },
-    default: { name: 'ZORA', url: 'https://explorer.zora.co' },
+    etherscan: { name: 'Explorer', url: 'https://explorer.zora.energy' },
```
Comment (tmm) [inline packages/chains/src/zora.ts:23]:
```suggestion
```

**Pull request merged**